docs/v4/objects/application.md

```diff
@@ -179,7 +179,7 @@ class ShutdownHandler
      * @param HttpErrorHandler  $errorHandler
      * @param bool              $displayErrorDetails
      */
-    public function __construct(Request $request, HttpErrorHandler $errorHandler, bool $displayErrorDetails) {
+    public function __construct(ServerRequestInterface $request, HttpErrorHandler $errorHandler, bool $displayErrorDetails) {
```

Comment:
Instead we should change the `use` statement to use the alias since we use that pattern in the docs:
`use Psr\Http\Message\ServerRequestInterface as Request;`

Reply:
Hi @l0gicgate 
Thank you for your feedback.
I' ll update this as per feedback.